docs/resources/authconfig.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: The authconfig Resource
+---
+
+# authconfig
+
+Use the `authconfig` InSpec audit resource to test the properties set in an authconfig file. The `authconfig` defines values that are used to track whether or not authentication mechanisms are enabled.
+
+## Syntax
+
+An `authconfig` resource block declares a authconfig file to be tested.  If no argument is given, the default file is set to '/etc/sysconfig/authconfig'.
+
+    describe authconfig do
+      its('USESSSD') { should eq 'yes' }
+    end
+
+## Supported Properties
+
+### installed>
```

Comment:
I totally understand the concern and I agree.  It will probably be better to test with authconfit --test because that is active in the sense that it is updated with any changes made but committed yet.  Ideally I think it would be good to have a resource that tests the active authconfig --test then and a subclass that tests the passive authconfig file.  The STIG has checks using both the file and command lines.